astropy/visualization/wcsaxes/core.py

```diff
@@ -451,10 +455,19 @@ def draw(self, renderer, inframe=False):
 
         self._drawn = True
 
-    def set_xlabel(self, label, labelpad=1, **kwargs):
+    # The label=None, xlabel=None (and similarly for ylabel) is to ensure
+    # compatibility with Matplotlib 2.x (which has label) and 3.x (which has
+    # xlabel). While these are meant to be a single positional argument,
+    # Matplotlib internally sometimes specifies e.g. set_xlabel(xlabel=...).
+
```

Comment:
Possibly still write `MATPLOTLIB_LT_30` here just for easier greping when that becomes the minimum supported version?